### PR TITLE
use asyncio_backport.get_running_loop and asyncio_backport.run on 3.6

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: uvicorn/_compat.py [hunk 1]
```diff
@@ -54,7 +88,7 @@ async def main():
                 loop.close()
 
     def _cancel_all_tasks(loop: AbstractEventLoop) -> None:
-        to_cancel = tasks.all_tasks(loop)  # type: ignore[attr-defined]
```
Comment (graingert) [inline uvicorn/_compat.py]: woops this was a fair cop from mypy

File: uvicorn/_compat.py [hunk 2]
```diff
@@ -5,74 +5,9 @@
...
-                continue
-            if task.exception() is not None:
-                loop.call_exception_handler(
-                    {
-                        "message": "unhandled exception during asyncio.run() shutdown",
-                        "exception": task.exception(),
-                        "task": task,
-                    }
-                )
+    from asyncio_backport import run, get_running_loop
```
Comment (graingert) [inline uvicorn/_compat.py]:
```suggestion
    from asyncio_backport import get_running_loop, run
```

File: uvicorn/_compat.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+https://github.com/python/cpython/blob/3.7/Lib/asyncio/runners.py
+"""
+import sys
+
+__all__ = ("run", "get_running_loop")
+
+if sys.version_info >= (3, 7):
+    from asyncio import get_running_loop, run
+else:
+    from asyncio_backport import run, get_running_loop
```
Comment (graingert) [inline uvicorn/_compat.py]:
```suggestion
    from asyncio_backport import get_running_loop, run
```

**Comment by Kludex** [comment]
We don't support Python 3.6 anymore.

**Pull request closed without merging**